arch/xtensa/src/esp32/esp32_cpuint.c

```diff
@@ -179,9 +179,17 @@ static uint32_t g_intenable[1];
  * devices.
  */
 
-static uint32_t g_cpu0_freeints = EPS32_CPUINT_PERIPHSET;
+static uint32_t g_cpu0_freeints = EPS32_CPUINT_PERIPHSET
+#ifdef CONFIG_ESP32_WIRELESS 
+                                  & (~(1 << ESP32_CPUINT_MAC))
+#endif
+                                  ;
```

Comment:
```suggestion
#ifdef CONFIG_ESP32_WIRELESS 
static uint32_t g_cpu0_freeints = EPS32_CPUINT_PERIPHSET & (~ (1  <<  ESP32_CPUINT_MAC));
#else
static uint32_t g_cpu0_freeints = EPS32_CPUINT_PERIPHSET;
#endif
```
I think this way it gets easier to understand. What do you think?